pkg/lca/imagebasedupgrade.go

```diff
@@ -273,6 +273,21 @@ func (builder *ImageBasedUpgradeBuilder) WithOadpContent(
 	return builder
 }
 
+// AutoRollbackOnFailureDisableForPostReboot allows controlling
+// AutoRollback on failure for post reboot stage. Enabled by default.
+func (builder *ImageBasedUpgradeBuilder) AutoRollbackOnFailureDisableForPostReboot(
```

Comment:
Please change function name to WithAutoRollbackOnFailureFlag.

Reply:
How about `WithRollbackDisabledForPostRebootConfig` ? I think we can just remove auto because there isn't anything in the spec not related to auto.

Reply:
disclaimer - I spent some time changing the function names yesterday.
The reason I went with AutoRollbackOnFailure in front because of this:
`RESOURCE: autoRollbackOnFailure <Object>

DESCRIPTION:
     <empty>

FIELDS:
   disabledForPostRebootConfig  <boolean>

   disabledForUpgradeCompletion <boolean>

   disabledInitMonitor  <boolean>

   initMonitorTimeoutSeconds    <integer>
`
Thus there will be 3 more functions for AutoRollbackOnFailure.
WDYT?